### PR TITLE
Added user manager and user table.

### PR DESCRIPTION
**Pull request by rohhan**

### CODE REVIEW

File: src/app/guildmanager/guildmanager.module.ts [hunk 1]
```diff
@@ -10,13 +10,14 @@ import { SidenavComponent } from './components/sidenav/sidenav.component';
...
 import { GuildTableComponent } from './components/guildtable/guildtable.component';
+import { UserManagerAppComponent } from '../usermanager/usermanagerapp.component';
 
 const routes: Routes = [
-  { path: '', component: GuildManagerAppComponent,
+  { path: 'guilds', component: GuildManagerAppComponent,
     children: [
       { path: '', component: MainContentComponent }
-    ] },
-  { path: '**', redirectTo: '' }
```
Comment (tgolia) [inline src/app/guildmanager/guildmanager.module.ts:19]: Why is this removed here? I remember this from the PluralSight course, but forget the use case..

File: src/app/models/user.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export interface IUser {
+  username: string;
```
Comment (tgolia) [inline src/app/models/user.ts:2]: Do we not want any of the other User properties? IsBot? DiscordId?

Comment (tgolia) [reply src/app/models/user.ts:2]: Or is the idea we only add them if they are being used (as opposed to parity with WebApi)?

File: src/app/services/user.service.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import { Observable } from 'rxjs';
+
+import { IUser } from '../models/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  private usersUrl = 'https://localhost:44399/api/users';
```
Comment (tgolia) [inline src/app/services/user.service.ts:12]: Why this port instead of default 4200?

File: src/app/guildmanager/guildmanager.module.ts (hunk 1 above)
Comment (rohhan) [reply src/app/guildmanager/guildmanager.module.ts:19]: The use case is a "catch all" for any undefined routes.  I was having some problems triggered the correct routes because the tutorial I followed used "nested routing modules" aka the main module has one router, and the feature modules have their own routers as well...Tbh it confused me...I should clean it up or learn exactly how it's working.

File: src/app/models/user.ts (hunk 2 above)
Comment (rohhan) [reply src/app/models/user.ts:2]: You are right.  We would have more properties based on what we want to use.  This was just to test the connection.  We don't need to have ALL the properties on the back end, just what we want to display.

File: src/app/services/user.service.ts (hunk 3 above)
Comment (rohhan) [reply src/app/services/user.service.ts:12]: i think 4200 is for the front end (angular) and this port was for the back end (apis)?

**Review by alltherobots — APPROVED** [review]

**Review by tgolia — APPROVED** [review]

**Review by tgolia — APPROVED** [review]
